### ISSUE TITLE
[feature request] on view unread add navigation

### ISSUE THREAD

**Issue by dwallace06** [mamaral/xkcd-Open-Source#68]
When on view unread it doesn’t seem to have the same arrow/swipe navigation (or I’m doing it wrong)

**Comment by mamaral** [comment]
You're correct, the current implementation doesn't do this but in my longer-term refactor it will. Thanks!
